include/swift/AST/ClangModuleLoader.h

```diff
@@ -40,6 +43,66 @@ enum class ClangTypeKind {
   ObjCProtocol,
 };
 
+/// A path for serializing a declaration.
+class StableSerializationPath {
+public:
+  struct ExternalPath {
+    enum StepKind {
+      /// A struct/union/class
+      Struct,
```

Comment:
What do you think about renaming this to `Record` to match the Clang name? I see that most of the other ones (all of them?) match the names for the corresponding Clang decls.

Reply:
Good idea.